services/test-data/dataset_defs/msg_fulldisk_ch2_netcdf.json

```diff
@@ -0,0 +1,47 @@
+{
+  "properties": {
+    "id": null,
+    "name": "MSG",
+    "description": "Meteosat Second Generation Channel 2",
+    "sourceOperator": "GdalSource"
+  },
+  "metaData": {
+    "GdalMetaDataRegular": {
+      "resultDescriptor": {
+        "dataType": "I16",
+        "spatialReference": "SR-ORG:81",
+        "measurement": "unitless"
+      },
+      "params": {
+        "filePath": "NETCDF:\"operators/test-data/raster/msg_netcdf/W_XX-EUMETSAT-Darmstadt,VIS+IR+HRV+IMAGERY,MSG4+SEVIRI_C_EUMG_%%%_START_TIME_%%%10.nc\":ch2",
```

Comment:
You haven't checked-in the data.
Is it possible to have a test data set?

Reply:
nope the data is not really public. the def is now here: https://github.com/geo-engine/dataset-defs/blob/main/msg_seviri_netcdf_ch2.json